homework/Lesson_19.py/phonebook_test.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import json
+from Lesson_26 import app_run, add_new, delete, update, search_by_first_name
+
+@pytest.fixture
+def contacts():
+    phonebook = 'contactstest.json'
+    data = {'contacts': [{'first name': 'Yura', 'last name': 'Hradiuk',
+                         'phone number': '067979797', 'city': 'Lviv'}]} 
+    with open(phonebook, 'w') as file:
+        json.dump(data, file)
+    yield phonebook  
+    with open(phonebook, 'w') as file:
+        json.dump(data, file)
```

Comment:
You could have just cleaned it as well. Every test that uses this fixture will set it up for itself (phonebook would be created). In other words, you do the same work (i.e., duplicate work) on lines 11 and 14.